client/src/utils/functions/multiSelectFunction.ts

```diff
@@ -0,0 +1,11 @@
+import { IOption } from '@interfaces/types';
+
+export const parserOption = (values: string, labels: string) => {
+    const valuesArr = values?.split(',');
+    const labelsArr = labels?.split(',');
+    const optionsArr: IOption[] = [];
+    for (let i = 0; i < valuesArr?.length; i++ ) {
+        optionsArr.push({ value: valuesArr[i], label: labelsArr[i] });
+    }
```

Comment:
I think that for dont need in a most of js issues, this issue same as most of all. Should use map